### PR TITLE
Changes in README

### PR DESCRIPTION
**Pull request by chicken-biryani**
I made the following changes:
1) Reduced the number of dead links in the FAQ section.
2) Added FLINT-example video link in the instruction set.
3) Capitalized the initials wherever it was required.
4) Changed the wording a little.

Note: I did not touch the 'instructions' part of the README since I am not well versed with it yet.

This PR deals with issue - #101

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,34 @@
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
 
-## What is `FLINT`?
+## What is FLINT?
 
-The Full Lands Integration Tool (FLINT) is a C++ framework that combines satellite and ground data in ways that meet policy needs. It is based on over 20 years of experience building and operating integration tools in Australia and Canada.   It's an Open-source Library maintained under [**moja.global**](https://github.com/moja-global/About-moja-global/blob/master/README.md), a project under the Linux Foundation
+Full Lands Integration Tool (FLINT) is a C++ framework that combines satellite and ground data  in a way that meets policy requirements.It is based on more than 20 years of experience in building and operating integrated tools in Australia and Canada. It's an Open-Source Library maintained under [**moja global**](https://github.com/moja-global/About-moja-global/blob/master/README.md), a project under the Linux Foundation.
```
Comment (HarshCasper) [inline README.md:7]: There is an extra space between `data` and `in`. Use a Markdown beautifier tool to mitigate these issues.

Comment (HarshCasper) [inline README.md:7]: There should be a space between the full stop and `It is based on`

Comment (HarshCasper) [inline README.md:7]: Add a hyperlink to Linux Foundation.

File: README.md [hunk 2]
```diff
@@ -145,17 +150,17 @@ Configuration file options:
...
-    * [submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](mojaglobal.slack.com). 
-* If you have other questions, please write to info@moja.global   
+* You can find FAQs on the [FAQs section of our docs](https://docs.moja.global/en/latest/faq.html).  
+* If you have a question about the code, submit a [user feedback](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) on the relevant repository  
+* If you have a general question about a project or repository or moja global, [join moja global](https://docs.moja.global/en/latest/contact.html) and:
+    * [Submit a discussion](https://help.github.com/en/articles/about-team-discussions) to the project, repository or moja global [team](https://github.com/orgs/moja-global/teams)
+    * [Submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](https://app.slack.com/client/T1G1M5HPF/slack-connect).
```
Comment (HarshCasper) [inline README.md:162]: The Slack link is wrong. It's `mojaglobal.slack.com`. We will soon have a more persistent Slack invite.

File: README.md [hunk 3]
```diff
@@ -145,17 +150,17 @@ Configuration file options:
 
 ## How to Get Involved?  
 
-moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and in the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).  
+moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and in the [About moja global Wiki](https://github.com/moja-global/.github/wiki).
```
Comment (HarshCasper) [inline README.md:154]: Remove the extra blank line

File: README.md [hunk 4]
```diff
@@ -2,29 +2,34 @@
...
++ Development managed using a true open source approach under [moja global](http://moja.global), which will allow users (countries, companies and organizations) to direct strategy and control the budget.
++ Software that allows data processing on local desktops or cloud-based systems.  
 
 
-## How to use FLINT?  
+## Getting Started with FLINT
+
+Below are instructions for quick installation.
+
+Note: If you need more detailed information, you can use a [step-by-step guide](https://github.com/moja-global/FLINT/tree/master/How_to_use_FLINT) or view the [video](https://www.youtube.com/watch?v=Jfi2-vEhfkg).
```
Comment (HarshCasper) [inline README.md:28]: Put `Note` in bold

File: README.md [hunk 5]
```diff
@@ -2,29 +2,34 @@
...
+
+Please provide us with feedback on how to improve these guidelines.
+You can leave your comment in the [version under development.](https://github.com/moja-global/FLINT/blob/master/How_to_use_FLINT/How_to_use_FLINT.md)
```
Comment (HarshCasper) [inline README.md:31]: Wrap both in a single line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (afzal442) [reply README.md]: We can also use this format as well `**_NOTE:_**`  as 

**_NOTE:_** If you need more detailed information ....

**Review by aornugent — APPROVED** [review]
Thanks @chicken-biryani

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**